src/benchmark_runner.h

```diff
@@ -42,6 +47,54 @@ struct RunResults {
   bool file_report_aggregates_only = false;
 };
 
+class BenchmarkRunner {
+ public:
+  BenchmarkRunner(const benchmark::internal::BenchmarkInstance& b_,
+                  std::vector<BenchmarkReporter::Run>* complexity_reports_);
```

Comment:
style guide has changed. `complexity_reports` is non-optional so should be a non-const reference (https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs).. you can change here or we can do it in another PR.

Reply:
This can not be a reference.